README.md

```diff
@@ -1,5 +1,7 @@
 <img src="https://user-images.githubusercontent.com/65091914/142253036-074f454e-62ab-445d-a30c-26e371e76e0a.png">
-Source: https://app.dbdesigner.net/designer/schema/0-codeheroes-99ad6258-244f-484c-a12f-151cb4aec1b2
+Source: https://app.dbdesigner.net/designer/schema/0-codeheroes-99ad6258-244f-484c-a12f-151cb4aec1b2  ===> `old table`
+
+Updated table Source: https://app.dbdesigner.net/designer/schema/480102
```

Comment:
Link re-directs me also to just https://app.dbdesigner.net/designer

Reply:
that's so weird can you guys try this address [https://dbdesigner.page.link/WTZRbVeTR7EzLvs86](https://dbdesigner.page.link/WTZRbVeTR7EzLvs86)

Reply:
that one works now thanks so much